fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -5564,6 +5564,10 @@ public ParseNode visitAggregationFunctionCall(StarRocksParser.AggregationFunctio
             isDistinct = context.aggregationFunction().setQuantifier().DISTINCT() != null;
         }
 
+        if (isDistinct && CollectionUtils.isEmpty(context.aggregationFunction().expression())) {
```

Comment:
distinct agg func has empty args? can this happen?

Reply:
```| COUNT '(' (setQuantifier bracketHint?)? (expression (',' expression)*)? ')'```
This lead `count(distinct )` is a valid clause but actually not.